docs/0.7/howto/state_delta_websockets.md

```diff
@@ -0,0 +1,76 @@
+# State Delta Subscriptions via Web Sockets
+
+<!--
+  Copyright 2022 Cargill Incorporated
+  Licensed under Creative Commons Attribution 4.0 International License
+  https://creativecommons.org/licenses/by/4.0/
+-->
+
+As transactions are committed to Scabbard, an app developer may be interested
+in receiving events related to the changes in state that result. These events,
+*StateDeltaEvents*, include information about the advance of the distributed
+ledger, as well as state changes that can be limited to specific address spaces
+in the global state.
+
+An application can subscribe to receive these events via a web socket,
+provided by the REST API component.
+
+# Opening a Web Socket
+
+The application developer must first open a web socket.
+
+Connect to the websocket at the following URL:
+```
+/{circuit_id}/{service_id}/ws/subscribe?last_seen_event={last_event_id}
```

Comment:
This begins with `/scabbard`

Reply:
Good catch. I even wrote out an app to test the ws subscribe action and somehow missed that part of the url. Fixed!